sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/autorest.md

```diff
@@ -6,7 +6,8 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 azure-arm: true
 namespace: Azure.ResourceManager.DeviceUpdate
-require: https://github.com/Azure/azure-rest-api-specs/blob/34018925632ef75ef5416e3add65324e0a12489f/specification/deviceupdate/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/32143b0f5f230ee2601e3c5d1990188666a5058d/specification/deviceupdate/resource-manager/readme.md
+tag: package-2020-03-01-preview
```

Comment:
Why not use the latest tag: package-2022-04-01-preview?

Reply:
I tried to update, but it seems `2022-04-01-preview` is not deployed to the production and the tests failed.